test/NuGet.Core.Tests/NuGet.Shared.Tests/NuGet.Shared.Tests.csproj

```diff
@@ -14,6 +14,9 @@
   <ItemGroup>
     <Compile Include="$(SharedDirectory)\*.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\TestUtilities\Test.Utility\Test.Utility.csproj" />
+  </ItemGroup>
```

Comment:
`Test.Utility` is a big, heavy assembly/project, where we have given up on the single responsibility principal and clean design. Can `XmlUtilityTests` have small modifications to no longer need anything from `Test.Utility`?

Reply:
@zivkan - sorry, I forgot to respond to this comment. I need only [`TestDirectory.cs`](https://github.com/NuGet/NuGet.Client/blob/dev/test/TestUtilities/Test.Utility/TestDirectory.cs)and [`TestFileSystemUtility.cs`](https://github.com/NuGet/NuGet.Client/blob/dev/test/TestUtilities/Test.Utility/TestFileSystemUtility.cs) classes to create and dispose test directory if needed. I see copying these 2 file to `NuGet.Shared.Tests` project as one option but not sure if we want to go in that direction.